src/GasPriceOracle.ts

```diff
@@ -44,15 +44,15 @@ export class GasPriceOracle {
     private static _instance: GasPriceOracle;
     private static asyncPoll: AsyncPoll;
 
-    private _gasPrice: number;
-    private readonly web3;
+    private _gasPrice;
+    private readonly ethereumService: EthereumService;
     private readonly gasPriceMetrics: GasPriceOracleMetrics;
 
     private constructor() {
-        this.web3 = new Web3(GETH_NODE);
+        this.ethereumService = new EthersEthereumService();
```

Comment:
Do we use `this.ethereumService` for anything other than utils in the GPO? It feels like instead of instantiating EthersEthereumService and saving it as a private member variable on the GPO, we could just import `ethers` and use the static utils functions, `ethers.utils.unitToWei`, etc.

Reply:
Or, after seeing the implementation, could we make it static and do something like `EthersEthereumService.unitToWei()`?

Reply:
We do utilize the instance for more than the static `unitToWei` method.  We are interacting with the provider to get the node's guess at the current best gas price.